spec/support/features/steps/cases/edit_steps.rb

```diff
@@ -186,8 +186,16 @@ def edit_ico_case_closure_step(kase:, decision_received_date: Date.today, ico_de
   expect(cases_show_page).to be_displayed(id: kase.id)
   expect(cases_show_page.notice.text)
     .to eq 'You have updated the closure details for this case.'
-  # expect(cases_show_page.case_details.response_details.whatever_al_calls_this_field.data.text)
+  # uncomment when this section is added to the page
+  # expect(cases_show_page.ico.case_details.response_details.date_ico_decision_received.data.text)
```

Comment:
This is not in the current designs and the only place this is played back is further up the page where it shows what the final decision was and the date it was received "ICO decision section"

So we might as well remove this commented out text

Reply:
Fair enough, I added it in though because I saw it on one of the designs.